aiida_testing/mock_code/_fixtures.py

```diff
@@ -123,6 +123,10 @@ def _get_mock_code(
         # can not be reused.
         code_label = f'mock-{label}-{uuid.uuid4()}'
 
+        data_dir_abspath = pathlib.Path(data_dir_abspath).absolute()
```

Comment:
Hmm, I'm not entirely sure we should be converting to an absolute path _here_. The reason I wanted `data_dir_abspath` to be _given_ as an absolute path is because resolving the path to absolute here will change depending on where the tests are executed from. For example, ``pytest`` from the project root will not do the same as ``cd tests; pytest``.

Maybe a better choice would be to _check_ that the path is absolute?

Reply:
Absolutely. Fixed.